aspnetcore/performance/caching/distributed.md

```diff
@@ -1,146 +1,177 @@
+title: Distributed caching in ASP.NET Core
+author: guardrex
+description: Learn how to use an ASP.NET Core distributed cache to improve app performance and scalability, especially in a cloud or server farm environment.
 ms.author: riande
 ms.custom: mvc
-ms.date: 02/14/2017
+ms.date: 10/19/2018
 uid: performance/caching/distributed
 ---
-# Work with a distributed cache in ASP.NET Core
+# Distributed caching in ASP.NET Core
 
-By [Steve Smith](https://ardalis.com/)
+By [Steve Smith](https://ardalis.com/) and [Luke Latham](https://github.com/guardrex)
 
-Distributed caches can improve the performance and scalability of ASP.NET Core apps, especially when hosted in the cloud or a server farm.
+A distributed cache is a cache shared by multiple app servers, typically maintained as an external service to the app servers that access it. A distributed cache can improve the performance and scalability of an ASP.NET Core app, especially when the app is hosted by a cloud service or a server farm.
 
-[View or download sample code](https://github.com/aspnet/Docs/tree/master/aspnetcore/performance/caching/distributed/sample) ([how to download](xref:tutorials/index#how-to-download-a-sample))
-
-## What is a distributed cache
-
-A distributed cache is shared by multiple app servers (see [Cache Basics](memory.md#caching-basics)). The information in the cache isn't stored in the memory of individual web servers, and the cached data is available to all of the app's servers. This provides several advantages:
-
-1. Cached data is coherent on all web servers. Users don't see different results depending on which web server handles their request
-
-2. Cached data survives web server restarts and deployments. Individual web servers can be removed or added without impacting the cache.
-
-3. The source data store has fewer requests made to it (than with multiple in-memory caches or no cache at all).
-
-> [!NOTE]
-> If using a SQL Server Distributed Cache, some of these advantages are only true if a separate database instance is used for the cache than for the app's source data.
-
-Like any cache, a distributed cache can dramatically improve an app's responsiveness, since typically data can be retrieved from the cache much faster than from a relational database (or web service).
-
-Cache configuration is implementation specific. This article describes how to configure both Redis and SQL Server distributed caches. Regardless of which implementation is selected, the app interacts with the cache using a common `IDistributedCache` interface.
+A distributed cache has the following advantages over other caching scenarios where cached data is stored on individual app servers:
 
-## The IDistributedCache Interface
+* Cached data is *coherent* (consistent) across requests to multiple servers.
+* Cached data survives server restarts and app deployments.
```

Comment:
Another reason to use it: cached data doesn't take local memory. (using an external cache)